yara/yara3/Makefile.acr

```diff
@@ -41,7 +41,7 @@ install:
 	${INSTALL_LIB} ${MOD}.${LIBEXT} $(R2PM_PLUGDIR)/${MOD}.${LIBEXT}
 	rm -rf $(R2PM_PLUGDIR)/yara
 	mkdir -p $(R2PM_PLUGDIR)/yara
-	cp -f ../rules-yara3/* $(R2PM_PLUGDIR)/rules-yara3
+	cp -rf ../rules-yara3/ $(R2PM_PLUGDIR)/
```

Comment:
I've just re-checked it and should be fine, since it's copying to `$(R2PM_PLUGDIR)` not to `$(R2PM_PLUGDIR)/rules-yara3`.